### DISCUSSION TITLE
how to use GAUGE_CUSTOM_BUILD_PATH for java project

### DISCUSSION THREAD

**Discussion by preetipadalia** [getgauge/gauge#2231] [General]
Hi,

I have a java gradle project. I am planning to convert it to a single jar to run the test. How can i use variable GAUGE_CUSTOM_BUILD_PATH  to achieve it?

Thanks

**Comment by sriv** [comment]
If you have the command to build the jar, you can set it to output to a location. This location can then be set as GAUGE_CUSTOM_BUILD_PATH value. gauge will then use the compiled jar for test execution

**Reply by preetipadalia** [reply]
Thank you @sriv . I will try and let you know.
